config/environments/development.rb

```diff
@@ -43,6 +39,9 @@
 
   config.cache_store = :dalli_store, nil, { value_max_bytes: 2.megabytes }
 
+  # Raises error for missing translations
+  config.action_view.raise_on_missing_translations = true
```

Comment:
No no! Please don't! We want graceful fallbacks in production, so this should be set to false. Probably a good idea to set it explicitly.